assembly_code/chp1_21.asm

```diff
@@ -0,0 +1,12 @@
+[org 0x0100]
+
+mov ax, 6  ; initialize the accumulator to 6
+add ax, 6  ; Adding 6 to ax
+add ax, 6  ; Adding 6 to ax
+add ax, 6  ; Adding 6 to ax
+add ax, 6  ; Adding 6 to ax
+add ax, 6  ; Adding 6 to ax
+; We only perform the add instruction 6 times because we initialized ax with 6
```

Comment:
```suggestion
; We only perform the add instruction 5 times because we initialized ax with 6
```

@Nabeegh-Ahmed @PySaqib @farhana1i please review EACH LINE of the questions you commit or have committed. There should even be no minor issues in the code. Appreciated! :)